src/d3a/d3a_core/sim_results/cumulative_grid_trades.py

```diff
@@ -15,16 +15,40 @@
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
-from d3a.d3a_core.sim_results.area_statistics import accumulate_grid_trades_all_devices, \
-    generate_cumulative_grid_trades_for_all_areas
-from d3a_interface.constants_limits import ConstSettings
+from d3a.constants import FLOATING_POINT_TOLERANCE
+from d3a.d3a_core.util import area_name_from_area_or_iaa_name, add_or_create_key, \
+    make_iaa_name_from_dict, subtract_or_create_key, round_floats_for_ui
 
 
-def export_cumulative_grid_trades(area, accumulated_trades_redis, past_market_types):
-    accumulated_trades = \
-        accumulate_grid_trades_all_devices(area, accumulated_trades_redis, past_market_types)
-    return accumulated_trades, generate_cumulative_grid_trades_for_all_areas(accumulated_trades,
-                                                                             area, {})
+def _is_cell_tower_type(area):
```

Comment:
you can now probably delete the corresponding `_is_cell_tower` method in the area_statistics.py, right?

Reply:
And of course for the other types too, if they are not needed somewhere else.

Reply:
https://github.com/gridsingularity/d3a/pull/875#discussion_r493424610